### PR TITLE
Generate code to parse protobuf

### PR DESCRIPTION
**Pull request by ahamez**
It removes the requirement to have protoc installed when using AxonONNX.
However, protoc is still required when/if code needs to be generated.

An alias is also provided (`mix generate_protobuf`) to make it easier to regenerate the code if needed.

Also, bumps protox version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,13 @@ def deps do
...
-- `protoc >= 3.0`. It must be installed in your system and available in your
-  `$PATH`. *This dependency is only required at compile-time*.
-  👉 You can download it [here](https://github.com/google/protobuf) or you can
-  install it with your favorite package manager (`brew install protobuf`,
-  `apt install protobuf-compiler`, etc.).
+### Protobuf
+
+Additionally, AxonONNX uses [protox](https://github.com/ahamez/protox) to generate code for parsing protocol buffers data within ONNX.
+
+If you want to regenerate this code, you can invoke the following command:
```
Comment (josevalim) [inline README.md:21]:
```suggestion
If you want to regenerate this code, you can invoke the following command:

```

**Comment by josevalim** [comment]
Thank you @ahamez, this is very nice!

Since we are pre-generating those, we no longer need the .proto files in `priv`. We can move them to `scripts` or the project root or whatever is protocol-buffers convention. WDYT?

File: README.md (hunk 1 above)
Comment (ahamez) [reply README.md:21]: Fixed!

**Comment by ahamez** [comment]
Yes, moving it out of `priv` makes sense. As far as I know, there's no convention for protobuf as to where we should put `*.proto` files 🤔. I guess as those files are more definitions than scripts, we could put `onnx.proto` in a `definitions` folder if that's OK with you?

**Comment by josevalim** [comment]
I would avoid extra directories, but it is up for @seanmor5 to decide!

**Comment by ahamez** [comment]
OK! In the meantime, I'll move it to `scripts` anyway, I guess the location is not that important as long as it's not in the `priv` folder.

**Review by josevalim — APPROVED** [review]

**Comment by seanmor5** [comment]
This is great! Removing the dependency on protoc is awesome. Ignore the failing CI...I will address that this week. My vote for .proto file is in scripts :)

**Comment by ahamez** [comment]
So, I could not completely ignore the failing CI, as it was a (small) problem with protox 😅. So, it's just a matter of a missing newline at the end of each generated file. I fixed that in version [1.6.10](https://hex.pm/packages/protox/1.6.10).
Do you want me to open a new PR to fix that?

**Comment by seanmor5** [comment]
Yes please!

**Pull request merged**